src/trainer_see.c

```diff
@@ -321,6 +321,10 @@ static const union AnimCmd *const sSpriteAnimTable_Emotes[] =
     sSpriteAnim_Emotes10,
 };
 
+// TODO: Move these declarations into even_object_movement.h
+#define OBJ_EVENT_PAL_TAG_MAY 0x1110
+#define OBJ_EVENT_PAL_TAG_EMOTES 0x8003
```

Comment:
A TODO that needs to be addressed

Reply:
They were already moved in the followers PR, so I'm just deleting these from here